QgisModelBaker/tests/test_import.py

```diff
@@ -41,12 +43,19 @@ def setUpClass(cls):
         """Run before all tests."""
         cls.basetestpath = tempfile.mkdtemp()
 
+    def test_import_mssql(self):
+        uri = "DSN={dsn};UID={uid};PWD={pwd}"\
+            .format(dsn="testsqlserver",
+                    uid="sa",
+                    pwd="<YourStrong!Passw0rd>")
+        con = pyodbc.connect(uri)
```

Comment:
I guess it misses here the import test, I did not completed it.

Reply:
Sure, we'll be adding tests